src/prefect/core/task.py

```diff
@@ -174,7 +175,7 @@ def run(self, x, y):
             is provided, it should have signature `callable(**kwargs) -> str` and at write
             time all formatting kwargs will be passed and a fully formatted location is
             expected as the return value.  Can be used for string formatting logic that
-            `.format(**kwargs)` doesn't support
+            `.format(**kwargs)` supports.
```

Comment:
I think the intent of this docstring was to say, if you can't do it within `.format()`, you can pass a callable instead

Reply:
Ahhh